client/knowledge_base_search/src/containers/SearchContainer/styles.scss

```diff
@@ -13,13 +13,31 @@
 .search_container--section {
   &._full-width-row {
     background-color: color('teal', 40);
-    padding: 0;
+    padding: 0 0 2.5rem 0;
     box-shadow: 0 7px 9px -6px rgba(0,0,0,0.7);
   }
 
-  .view_all_button--div {
-    text-align: right;
-    padding: 1rem;
+  .training-key {
+    &--icon {
+      height: 0.7rem;
+      width: 0.7rem;
+      display: inline-flex;
+      margin-right: 0.5rem;
+      border: 1px solid color('white', 10);
+      border-radius: 50%;
+      background-color: color('purple', 50);
+    }
+
+    &--text {
+      color: color('teal', 80);
+      font-size: 0.75rem;
+      font-weight: 300;
```

Comment:
maybe see if you can use the typography settings defined in https://watson-developer-cloud.github.io/ui-components/style.html#typography

```
font-size: get('weight light');
```
maybe?